Add Sentry integrations to registry (#2236)
open-telemetry/opentelemetry.io: content/en/registry/tools-java-sentry.md

---
title: OpenTelemetry Java Integration for Sentry (sentry.io)
registryType: utilities
isThirdParty: true
language: java
tags:
  - java
  - processor
  - propagator
  - utilities
  - sentry
  - error monitoring
repo: https://github.com/getsentry/sentry-java/tree/main/sentry-opentelemetry
license: MIT

> description: The Sentry OpenTelemetry Java integration provides a processor and propagator to send data to Sentry and to associate traces/spans to Sentry errors. [See the Sentry docs to see how to configure](https://docs.sentry.io/platforms/java/performance/instrumentation/opentelemetry/).

Reviewer: ```suggestion
description: The Sentry OpenTelemetry Java integration provides a processor and propagator to send data to Sentry and to associate traces/spans to Sentry errors. For configuration details, see [OpenTelemetry Support](https://docs.sentry.io/platforms/java/performance/instrumentation/opentelemetry/).
```
